pkg/controller/certificates/sync.go

```diff
@@ -41,10 +39,19 @@ const (
 	messageIssuingCertificate  = "Issuing certificate..."
 	messageRenewingCertificate = "Renewing certificate..."
 
+	warningCertificateDuration = "WarnCertificateDuration"
+	warningScheduleModified    = "WarnScheduleModified"
+
 	messageCertificateIssued  = "Certificate issued successfully"
 	messageCertificateRenewed = "Certificate renewed successfully"
+
+	messageWarningCertificateDuration = "Certificate duration received from issuer is %s which is lower than the issuer configured duration of %s"
+	messageWarningScheduleModified    = "Certificate renewal requested schedule cannot be honored. Specified renewBefore of %s is greater than certificate total duration of %s"
```

Comment:
This sounds like a hard error to me - perhaps rephrase to explain that we've instead changed the renewal date?

I'd also make this a non-warning (i.e. EventTypeNormal), as some users will take a warning to be an error and starting opening issues 😬